examples/net_client/main.rs

```diff
@@ -72,32 +75,3 @@ impl<'a> System<'a> for SpamSystem {
         }
     }
 }
-
-/// A simple system reading received events.
-/// Used to see events sent by the net_echo_server example.
-struct ReaderSystem {
```

Comment:
We can't read received packets from `System`s now?

Reply:
Wasn't used so I figured to remove it. The net_server does not send packets back.

Reply:
I added it again